lms/djangoapps/teams/static/teams/js/views/team_profile_header_actions.js

```diff
@@ -9,7 +9,7 @@
         function (Backbone, _, gettext, TeamUtils, teamProfileHeaderActionsTemplate) {
             return Backbone.View.extend({
 
-                errorMessage: gettext("An error occurred. Try again."),
+                errorMessage: _.escape(gettext("An error occurred. Try again.")),
```

Comment:
@cahrens actually that example is because other _languages_ may contain non-HTML characters. Will Chinese, Arabic, macrons, etc all display properly here if they are provided by the translators?